examples/guides/animation/step-1.html

```diff
@@ -3,19 +3,13 @@
 
 <head>
     <!-- Include CARTO VL JS -->
-    <script src="../../../../dist/carto-vl.js"></script>
+    <script src="../../../dist/carto-vl.js"></script>
     <!-- Include Mapbox GL JS -->
     <script src="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.js"></script>
     <!-- Include Mapbox GL CSS -->
     <link href="https://libs.cartocdn.com/mapbox-gl/v0.48.0-carto1/mapbox-gl.css" rel="stylesheet" />
     <!-- Make the map visible -->
-    <style>
-        #map {
-            position: absolute;
-            height: 100%;
-            width: 100%;
-        }
-    </style>
+    <link rel="stylesheet" type="text/css" href="../../style.css">
```

Comment:
But I won't change this!

Examples inside guides are meant to be reproduced, step by step, by the users, and adding style.css dependency is more complex than just having a style rule for the map. In the guides I've worked that's enough.

Ping to @elenatorro here to agree on this before changing it

Reply:
Most maps I saw shown visual problems due to incorrect CSS (when the CSS was the inlined one), and I don't think it's possible to have a small CSS for many examples. At least in this way, the CSS doesn't add noise. We are trying to help people with CARTO VL, not with CSS.

Reply:
I would be very careful when changing `step-N.html` examples for the guides, as it usually also requires sync fixes in the content of the .md files

Reply:
We have agreed on going on with this fix, and delay changes & possible refactoring on css. We will try that in #1041